channeld/channeld.c

```diff
@@ -264,9 +264,20 @@ static const u8 *psbt_to_tx_sigs_msg(const tal_t *ctx,
 				     struct channel *channel,
 				     const struct wally_psbt *psbt)
 {
-	const struct witness_stack **ws =
-		psbt_to_witness_stacks(tmpctx, psbt,
-				       channel->opener);
```

Comment:
Since this will always be our sigs, we can use `our_tx_role`. (takes a peer, you can just pass the peer in to this instead of channel)

Reply:
Ok, will do tomorrow :)